README.md

```diff
@@ -1398,13 +1398,15 @@ This service enables a device to report accumulated consumption and/or pulses in
 
 #### Interfaces
 
-Type | Interface                      | Value type |  Props  | Description
------|----------------------------- --|------------|---------|------------
-in   | cmd.meter.set_interval         | int        |         | Interval in minutes for accumulated consumption recalculation. Overwrites a default value of 30 minutes.
-in   | cmd.meter.add                  | float_map  | `unit`  | Adds corresponding meter service (eg. meter_elec) to a selected device to report accumulated consumption. Map of floats shall provide consumption for every mode.
-in   | cmd.meter.remove               | null       |        | Removes all added virtual meter services from a selected device. The device shall not be reporting accumulated consumption nor pulses anymore.
+Type | Interface               | Value type | Description
```

Comment:
Props column should stay. Unit prop should be included in cmd.meter.add and cmd.meter.get_report and evt.meter.report to indicate which unit configuration you're setting and for which unit you request report and get it.

Reply:
@BudzikM As the documentation should reflect the reality. What is the status of the `unit` prop in all of those commands? I reverted back one for `cmd.meter.add` because it was already there, but what with the rest?

Reply:
@BudzikM you got your answer from Bugos ? Unit is a unit, we want to know what is measured.